test/inst_scc_test.rb

```diff
@@ -28,6 +28,16 @@
       expect(Yast::WFM.call("inst_scc")).to eq(:abort)
     end
 
+    it "displays an error when loading the available extensions fails" do
```

Comment:
Well, strictly speaking you are not checking that the error is displayed. You are simply checking that the whole thing does not explode... which is actually nice to know (and proves the bug to be fixed) but not what the predicate says. :-)